core/uploader.go

```diff
@@ -46,57 +46,33 @@ var expiryField = map[string]string{
 	"Object-Expires": "+168h", // Objects will be deleted after 7 days
 }
 
-func Upload(input io.Reader, outputURI *url.URL, waitBetweenWrites, writeTimeout time.Duration) (*drivers.SaveDataOutput, error) {
-	output := outputURI.String()
-	storageDriver, err := drivers.ParseOSURL(output, true)
-	if err != nil {
-		return nil, err
-	}
-	session := storageDriver.NewSession("")
-	if err != nil {
-		return nil, err
-	}
-
-	// While we wait for storj to implement an easier method for global object deletion we are hacking something
-	// here to allow us to have recording objects deleted after 7 days.
-	fields := &drivers.FileProperties{}
-	if strings.Contains(output, "gateway.storjshare.io/catalyst-recordings-com") {
-		fields = &drivers.FileProperties{Metadata: expiryField}
-	}
+type StorageBackupURLs []struct {
```

Comment:
WDYT of `StorageFallbackURLs`, would that be clearer? At least it refers to the act of falling back from primary to the backup. Just `StorageURLs` doesn't seem accurate either, since one doesn't need to specify this config for every storage that is used, only for the ones that a backup storage is desired.